index.html

```diff
@@ -710,21 +710,12 @@ <h2>Trust Model</h2>
 this specification regarding syntax. That is, the content in the
 <a href="#basic-concepts">Basic Concepts</a>,
 <a href="#advanced-concepts">Advanced Concepts</a>, and
-<a href="#syntaxes">Syntaxes</a> sections of this document. For convenience,
-normative statements for <em>conforming documents</em> are
-often phrased as statements on the syntax used in properties and their
-associated values in the document (for example, "MUST be a URI", or
-"MUST be a string value of an ISO8601 combined date and time string").
+<a href="#syntaxes">Syntaxes</a> sections of this document.
       </p>
-
+      
       <p>
-A <em>conforming processor</em> is a software or hardware-based implementation
-of the normative statements in this specification regarding the expected
-contents of property-value pairs (for example, the content in
-<a>Verification</a>). For convenience, normative statements for
-<em>conforming processors</em> are often phrased as behavioral statements
-regarding the contents of property-value pairs (for example, "MUST NOT be
-revoked", or "MUST be in the expected range").
+This document makes no normative statements with regard to conformance of
```

Comment:
Hmm, I think this is wrong. We need to make a statement about "conforming document processors". I believe you have identified "conformant behavior for roles" in the system, which the spec shouldn't say anything about.

I'm going to pull the PR in, but modify it to effectively state what I think you're going for (and thus it will be an editorial change).